grakn-migration/csv/src/main/java/ai/grakn/migration/csv/Main.java

```diff
@@ -90,6 +92,12 @@ public static void runCSV(ClusterManager manager, CSVMigrationOptions options){
             die(throwable);
         }
 
-        initiateShutdown();
+        if (newClusterManager) {
+            try {
+                manager.stop();
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
+        }
     }
 }
```

Comment:
Something smells bad here if a migrator needs to start an Engine ClusterManager. Why is that?